web/src/app/AccountApi.js

```diff
@@ -369,6 +369,7 @@ class AccountApi {
       if (e instanceof UnauthorizedError) {
         session.resetAndRedirect(routes.login);
       }
+      return undefined;
```

Comment:
Why is this required?

Reply:
The rule is consistent returns - `undefined` was already implied as the default return type, but since other branches do explicitly return values, it wants you to also return explicitly here too.